packages/endpoint-copier-operator/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/suse-edge/endpoint-copier-operator.git
```

Comment:
Do you think we'll want to keep the chart in the endpoint-copier-operator repo long term, or move it into a package here similar to the other components?

Reply:
Keeping the chart in the `endpoint-copier-operator` repo was suggested in our Slack channel and the arguments were that in this way we keep the repo completed.
Akri follows the same approach.